File: CS/BusinessClasses/PersistentClasses.cs
using DevExpress.Xpo;

namespace DXSample.PersistentClasses {
    public class Person : XPObject {
        public Person(Session session) : base(session) { }

        private string fName;
        public string Name {
            get {
                return fName;
            }
            set {
                SetPropertyValue("Name", ref fName, value);
            }
        }

        private int fAge;
        public int Age {
            get {
                return fAge;
            }
            set {
                SetPropertyValue("Age", ref fAge, value);
            }
        }
    }
}

File: VB/BusinessClasses/PersistentClasses.vb
Imports Microsoft.VisualBasic
Imports System
Imports DevExpress.Xpo

Namespace DXSample.PersistentClasses
	Public Class Person
		Inherits XPObject
		Public Sub New(ByVal session As Session)
			MyBase.New(session)
		End Sub

		Private fName As String
		Public Property Name() As String
			Get
				Return fName
			End Get
			Set(ByVal value As String)
				SetPropertyValue("Name", fName, value)
			End Set
		End Property

		Private fAge As Integer
		Public Property Age() As Integer
			Get
				Return fAge
			End Get
			Set(ByVal value As Integer)
				SetPropertyValue("Age", fAge, value)
			End Set
		End Property
	End Class
End Namespace